src/components/ProjectLabel.module.scss

```diff
@@ -0,0 +1,19 @@
+.label {
+  width: 100%;
+  text-align: left;
+  padding: 0 10px;
+  display: flex;
+  align-items: center;
+}
+
+.name {
+  color: var(--black);
```

Comment:
Old css-var colors are deprecated and there was a warning in discord. Please, use the naming of the up-to-date color scheme

Reply:
@DmitriVanGurad
black doesn't seems to be mapped with any new var. I've done the changes for other colors

Reply:
@devanshu-santiment If the color palette does not have a color with the corresponding name, just check hex color
![image](https://user-images.githubusercontent.com/25135650/55502049-626c0a00-5654-11e9-8d67-c0d820dc821a.png)